pkgs/by-name/ri/ripes/package.nix

```
@@ -25,17 +22,26 @@ stdenv.mkDerivation rec {
     hash = "sha256-aNJTM/s4GNhWVXQxK1R/rIN/NmeKglibQZMh8ENjIzo=";
   };
 
+  # Fixes the following error during build
+  # /tmp/nix-build-ripes-2.2.6-unstable-2024-04-04.drv-0/source/external/VSRTL/external/cereal/include/cereal/external/rapidjson/document.h:319:82: error: cannot assign to non-static data member 'length' with const-qualified type 'const SizeType' (aka 'const unsigned int')
```

```suggestion
  # external/VSRTL/external/cereal/include/cereal/external/rapidjson/document.h:319:82: error: cannot assign to non-static data member 'length' with const-qualified type 'const SizeType' (aka 'const unsigned int')
```
we can shorten that a bit

---

Sure, done